### DISCUSSION TITLE
multiple wake words for multiple languages

### DISCUSSION THREAD

**Discussion by muzzol** [OpenVoiceOS/ovos-buildroot#196] [Q&A] [answered]
Hi,

I'm pretty new to OVOS and I would like to use Catalan as the main language and fall back to english if something isn't working in catalan.

Can I configure multiple wake words so I go to catalan configuration with "**Bon dia OVOS**" and to english with "**Good morning OVOS**" ?

Is that a good aproach to use OVOS with two different languages?

thanks in advance

**Comment by JarbasAl** [comment, marked as answer]
Hello, apologies for the super late answer

I added a section to our docs just about this https://openvoiceos.github.io/ovos-technical-manual/speech_service/#hotwords

**Reply by muzzol** [reply]
thanks a lot for your time
